### DISCUSSION TITLE
Random/shuffle mode

### DISCUSSION THREAD

**Discussion by gerroon** [bpatrik/pigallery2#616] [General]
Hi

Does this have a way to show photos in randomized shuffle way from one or multiple galleries?

**Comment by bpatrik** [comment]
yes. you can short them by random. Please check the demo app: https://pigallery2.onrender.com/gallery/

**Reply by gerroon** [reply]
Thanks but it does not work properly. In the album page I click on an album, then I choose random. Now it does random once, when I want to randomize once more I need to reload the page again and pick the random again. It seems not user friendly in that way.
